src/apps/popup/hooks/use-account-actions-with-events.ts

```diff
@@ -56,16 +56,15 @@ export function useAccountManager() {
   const isLocked = useSelector(selectVaultIsLocked);
   const activeAccount = useSelector(selectVaultActiveAccount);
   const accounts = useSelector(selectVaultAccounts);
-  const accountNamesByOriginDict = useSelector(
-    selectVaultAccountNamesByOriginDict
-  );
+  const accountNamesByOriginDict = useSelector(selectAccountNamesByOriginDict);
```

Comment:
We can do this here and remove it in other places
```suggestion
  const accountNamesByOriginDict = useSelector(selectAccountNamesByOriginDict) || [];
```

Reply:
No, it's a dictionary, not an array.